eslintrc.json

```diff
@@ -15,8 +15,6 @@
 		"no-script-url": 2,
 		"no-shadow": 2,
 		"no-tabs": 0,
-		"no-template-curly-in-string": 0,
```

Comment:
I'd expect `ecmaVersion: 5` to disable es6 rules. If not, I think we should add the relaxing rules to the eslint config in Apps instead of this one.